frontera/logger/formatters/color.py

```diff
@@ -10,7 +10,7 @@
 class ColorFormatter(ColoredFormatter):
 
     def __init__(self, format, log_colors, log_color_field, datefmt=None, reset=True, style='%'):
-        super(ColorFormatter, self).__init__(format=format, datefmt=datefmt, log_colors=log_colors,
+        super(ColorFormatter, self).__init__(fmt=format, datefmt=datefmt, log_colors=log_colors,
```

Comment:
I do not think that this is a good idea. With a positional argument we might not even realize that the method changed and strange things could happen without us knowing why. But then again it's a matter of taste.